include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -163,7 +163,7 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
       set_byte_count(ByteCount() + slice_length);
       if (slice_length <= static_cast<uint64_t>(count)) {
         cord->Append(MakeCordFromSlice(grpc_slice_ref(*slice())));
-        count -= slice_length;
+        count -= static_cast<int>(slice_length);
```

Comment:
can you add a comment as to why this is safe?

Reply:
Resolved.